dataverse-persistence/src/main/resources/db/migration/V23__addVocabularyValuesGroupingAndSortingByLocalizedStrings.sql

```diff
@@ -0,0 +1,196 @@
+ALTER TABLE controlledvocabularyvalue 
+ADD COLUMN displaygroup text;
+
+DELETE FROM controlledvocabularyvalue
+WHERE datasetfieldtype_id = 35;
```

Comment:
We cannot relay on `id's` to be exactly the same on each installation. They are generated so instead of filtering on `id` use some `natural key` value like `strvalue`.

Reply:
The id's are not generated - fixed values inserted in v2__bazeSchema.sql